css-positioning/absolute.css

```diff
@@ -0,0 +1,44 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url("images/one-does-not-simply.jpg");
+  background-size: cover;
+
+  /* your code here */
+  position: relative;
+}
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+
+  /* your code here */
+  position: absolute;
+  right: 1rem;
+  top: 1rem;
+}
+
+.joke {
+  margin-bottom: 1rem;
+  font-size: 2rem;
+  font-family: Anton, sans-serif;
+  text-align: center;
+  color: white;
+  -webkit-text-stroke: 1px black;
+  text-transform: uppercase;
+
+  /* your code here */
+  position: absolute;
+  bottom: 0;
+}
+
+.overlay {
+  background: linear-gradient(transparent, rgb(0 0 0 / 75%));
+
+  /* your code here */
+  position: absolute;
+  bottom: 0;
```

Comment:
You'll need to also set the top, left, and right so it covers the whole picture.